src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -130,27 +134,27 @@ const PreReleaseUserAccessForm = ({
                 },
               })
               .test({
-                name: 'email format',
-                message: 'Enter only @education.gov.uk email addresses',
-                test: (value: string) => {
+                name: 'lines-contain-valid-emails',
+                message: ({ value }) =>
+                  `'${value}' is not a valid email address`,
+                test(value?: string) {
                   if (value) {
                     const emails = splitAndTrimLines(value);
-                    return emails.every(email => {
-                      if (
-                        /^simulate-delivered(?:-[1-3])?@notifications.service.gov.uk$/i.test(
-                          email,
-                        )
-                      ) {
-                        return true;
-                      }
-                      const emailSegments = email.split('@');
-                      return (
-                        emailSegments.length === 2 &&
-                        emailSegments[1] === 'education.gov.uk'
-                      );
+                    const indexOfFirstInvalid = emails.findIndex(
+                      email => !isValidEmail(email),
+                    );
+                    if (indexOfFirstInvalid < 0) {
+                      return true;
+                    }
+                    // eslint-disable-next-line react/no-this-in-sfc
+                    return this.createError({
+                      path: 'emails',
+                      params: {
+                        value: emails[indexOfFirstInvalid],
+                      },
                     });
                   }
-                  return false;
+                  return true;
                 },
               }),
```

Comment:
It seems like we should be able to use `context: Yup.TestContext` here as a parameter and use arrow syntax rather than `this` but I struggled to get this to work. I think it's fixed in a more recent version of `@types/yup` but upgrading that led to other type errors elsewhere.

Example:

```
                test: (context: TestContext, value?: string) => {
                  if (value) {
                    const emails = splitAndTrimLines(value);
                    const indexOfFirstInvalid = emails.findIndex(
                      email => !isValidEmail(email),
                    );
                    if (indexOfFirstInvalid < 0) {
                      return true;
                    }
                    return context.createError({
                      path: 'emails',
                      params: {
                        value: emails[indexOfFirstInvalid],
                      },
                    });
                  }
                  return true;
                },
```

Reply:
Shame about this, but I think you've done the right thing here to get it working. We should keep an eye out for when Yup v1 is ready and upgrade then.